doc/user_guide/local_installation.md

```diff
@@ -0,0 +1,100 @@
+## How to install the Exasol Connector for Power Apps.
```

Comment:
```suggestion
## How to Install the Exasol Connector for Power Apps
```

Reply:
We usually use Associated Press style headings (https://capitalizemytitle.com/style/APA/).